recipes/conda/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "4.0.6" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  fn: conda-{{ version }}.tar.gz
+  url: https://github.com/conda/conda/archive/{{ version }}.tar.gz
+  sha256: 4ca7ec875de13519aeb3eebc9beca23adf3c41e8852c186deb503e8bc2282c40
+
+build:
+  number: 0
+  script: CONDA_DEFAULT_ENV='' python setup.py install
+
+requirements:
+  build:
+    - python
+    - conda-env
+    - pycosat
```

Comment:
So, this is required for the solver. I'm pretty sure it should be there. Care to comment on this, @mcg1969?